shared/login/register/code-page/index.render.desktop.js

```diff
@@ -54,39 +57,32 @@ export default class CodePageRender extends Component {
 
   renderEnterText () {
     return (
-      <div style={{display: 'flex', flexDirection: 'column', flex: 1, backgroundColor: 'green', alignItems: 'center', justifyContent: 'center', padding: 20}}>
-        <h1>Type the verification code from your other device into here</h1>
-        <TextField
-          hintText='Type code here'
-          floatingLabelText='Code'
-          value={this.props.enterText}
+      <Container
+        style={styles.container}
+        onBack={this.props.onBack}>
+        <Text style={{marginTop: 38, marginBottom: 11}} dz2 type='Header'>Type in text code</Text>
+        {subTitle()}
+
+        <Icon style={{marginTop: 30, marginBottom: 40}} type='phone-text-code' />
+        <Input
+          dz2
+          style={{alignSelf: 'stretch'}}
+          fullWidth
+          hintText='opp blezzard tofi pando agg whi pany yaga jocket daubt bruwnstane hubit yas'
+          floatingLabelText='Text code'
           multiLine
+          value={this.props.enterText}
           onChange={event => this.props.onChangeText(event.target.value)}
         />
-        <RaisedButton
-          style={{alignSelf: 'flex-end', marginTop: 20}}
-          label='Submit'
-          primary
-          onClick={() => this.props.textEntered(codePageModeEnterText)}
-        />
-      </div>
+        <Button dz2 style={{alignSelf: 'flex-end', marginTop: 35, marginBottom: 20}} label='Continue' onClick={() => this.props.textEntered(codePageModeEnterText)} />
```

Comment:
Ah, now that dz2 Button's been merged into master, I'd expect this to fail Flow 'cause you don't have a `type='Primary'` or similar specified.  Try again on master?